libcob/ChangeLog

```diff
@@ -86,6 +121,19 @@
 	* fileio.c (apply_file_paths): extracted from cob_chk_file_mapping
 	  to factor out duplicated code
 
+2024-05-30  Chuck Haatvedt <chuck.haatvedt+cobol@gmail.com>
+
+		fix errors caught by the Sanitizer functionality of GCC. This
+		change did not resolve all of the issues found as some of them
+		were in Berkeley DB and some were intentional in the memory
+		corruption logic which has been implemented in the runtime.
```

Comment:
I guess that was reformatted with a later commit...

Reply:
Yes, by 5343 (which modifies more than this).